frontend/src/modules/lf/segments/components/fragments/lf-project-integration-column.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    v-if="props.integrations?.length"
+    class="flex gap-1 items-center py-2 overflow-x-auto"
+    style="max-width: 100%; scrollbar-width: thin"
+  >
+    <div v-for="integration in props.integrations" :key="integration.id">
+      <el-popover
+        :disabled="
+          integration.status !== 'in-progress'
+            && integration.type !== 'mapped'
+            && !getProgress(props.segmentId, integration.platform)
+            && !props.progressError
+        "
+        :width="290"
+        placement="top"
+      >
+        <template #reference>
+          <div
+            class="relative w-6 h-6 flex-shrink-0 flex items-center justify-center"
+          >
+            <app-platform-svg
+              :platform="integration.platform"
+              :color="integration.platform === 'github' && integration?.type === 'mapped' ? 'gray' : 'black'"
+            />
+
+            <lf-icon
+              v-if="integration.status === 'no-data'"
+              name="triangle-exclamation"
+              type="solid"
+              :size="12"
+              class="absolute right-0 top-0 leading-3 text-yellow-500"
+            />
```

Comment:
I believe you didn't update the styling of the connected platforms, can you update to match the designs?
![Screenshot 2024-12-04 at 08 27 50](https://github.com/user-attachments/assets/92348137-056d-4d19-aa55-10ba086ee82d)
<img width="675" alt="Screenshot 2024-12-04 at 08 28 25" src="https://github.com/user-attachments/assets/7b7ae4df-2913-4b6e-b695-077cd22555ec">

Reply:
done